server/src/config.rs

```diff
@@ -8,4 +9,15 @@ pub struct Config {
     pub address: url::Url,
     #[structopt(long, env = "NET_IP_CHECKER_PORT")]
     pub ip_checker_port: u16,
+    #[structopt(long, env, parse(try_from_str = humantime::parse_duration), default_value = "10s")]
+    pub session_cleaner_interval: Duration,
+    #[structopt(long, env, default_value = "60")]
+    pub session_timeout: i64,
+    #[structopt(long, env, default_value = "1073741824")]
+    pub forwarder_rate_limit: u32,
+    #[structopt(long, env, parse(try_from_str = humantime::parse_duration), default_value = "1s")]
+    pub forwarder_resume_interval: Duration,
 }
+
+// pub static ref FORWARDER_RATE_LIMIT: u32 = typed_from_env::<u32>("FORWARDER_RATE_LIMIT", 2048);
```

Comment:
should these still be added? maybe we can add a TODO if we want to keep them

Reply:
forgot to remove them